sksparse/cholmod.pyx

```diff
@@ -31,8 +31,6 @@
 
 #cython: binding = True
 #cython: language_level = 3
-#distutils: include_dirs = NP_GET_INCLUDE /usr/include/suitesparse
```

Comment:
I would completely strip out the distutils section here... the entry in `setup.py` is enough and having conflicting info is the last thing we want.